### DISCUSSION TITLE
Need to refer to the corresponding Human Interface Guidelines

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#298] [Show and tell]
I was thinking about why Apple.inc abandoned the Mac OS 9 classic theme, and suddenly thought that they must have a previous development guide with UI, so I found this as a reference for everyone.

**Aqua**
[https://archive.org/details/AquaHIGuidelinesPreliminary/mode/2up](https://archive.org/details/AquaHIGuidelinesPreliminary/mode/2up)

**Comment by probonopd** [comment]
Thanks for the link.

We are loosely following the principles laid out in https://dl.acm.org/doi/book/10.5555/573097. Those are written in the Classic era (before Aqua). https://asktog.com/atc/principles-of-interaction-design/ is also highly recommended.
